### PR TITLE
 feat(controllers): added controller for post /movies end point

### PR DESCRIPTION
**Pull request by alikslob**
Why:The route still had no functionality other than validating payload
What:
-adds controller for post .movies route 
-adds test for controller
-adds integration tests

### CODE REVIEW

File: test/plugins/features/movies/controller.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+describe('movie controller', () => {
+
+  describe('create', () => {
+
+    it('creates a movie', async () => {
+      const payload = { title: 'WALL-E' };
+
+      const movie = await Controller.create(payload);
+
+			      expect(movie.get('title')).to.eql(payload.title);
```
Comment (vinay-pad) [inline test/plugins/features/movies/controller.test.js]: Indentation looks weird here

**Review by vinay-pad — APPROVED** [review]
LGTM

File: test/validators/movie.test.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 'use strict';
 
-const Joi = require('joi');
-
+const Joi            = require('joi');
```
Comment (ajpawlicki) [inline test/validators/movie.test.js]: nit: I would leave the newline so we can separate external dependencies from internal files that we require and then we don't need to align these

File: lib/plugins/features/movies/index.js [hunk 3]
```diff
@@ -1,12 +1,15 @@
 'use strict';
+
 const MovieValidator = require('../../../validators/movie');
+const Controller     = require('./controller');
```
Comment (ajpawlicki) [inline lib/plugins/features/movies/index.js:3]: nit: alphabetize

**Review by ajpawlicki — APPROVED** [review]
LGTM

**Pull request merged**